buildSrc/src/main/java/org/opensearch/gradle/OpenSearchTestBasePlugin.java

```diff
@@ -198,6 +198,7 @@ public void execute(Task t) {
                 logging.setShowExceptions(true);
                 logging.setShowCauses(true);
                 logging.setExceptionFormat("full");
+                logging.setShowStandardStreams(System.getProperty("tests.output").equals("always"));
```

Comment:
Check if not null or supply a default value to the `getProperty`?

Reply:
How sloppy on my part, sorry. Updated to just be a boolean.